tests/bluetooth/tester/src/btp.c

```diff
@@ -31,11 +31,8 @@ static struct k_thread cmd_thread;
 #define CMD_QUEUED 2
 struct btp_buf {
 	intptr_t _reserved;
-	uint8_t rsp[BTP_MTU];
-	union {
-		uint8_t data[BTP_MTU];
-		struct btp_hdr hdr;
-	};
+	uint8_t data[BTP_MTU]; /* includes btp header */
+	uint8_t rsp[BTP_MTU - sizeof(struct btp_hdr)];
```

Comment:
Consider defining a `BTP_RSP_MTU` or something.

Reply:
RSP has same MTU, it is just that we don't have (need) header in response buffer (since data from command header is used)